### ISSUE TITLE
Reproduction issue: self-trained Qwen3-VL-4B-Instruct on LIBERO gets 0 success rate, while the released checkpoint works correctly

### ISSUE THREAD

**Issue by JahaoZhang** [starVLA/starVLA#352]
Hi,

Thanks for releasing StarVLA.

I am trying to reproduce the LIBERO training results following the official Quick Start Guide with Qwen3-VL-4B-Instruct.

I tested the released pretrained checkpoint provided by your project, and the evaluation result looks correct. Therefore, the LIBERO environment, evaluation script, and benchmark setup should be working properly on my side.

However, when I train the model myself using the official Quick Start configuration, the training loss decreases smoothly and the loss curve looks reasonable, but the final evaluation success rate is 0.

One small modification I made during training is in the action prediction part:

# Original
pred_actions = self.action_model.predict_action(action_queries)

# Modified
pred_actions = self.action_model.predict_action(action_queries.float())

I made this change due to dtype-related issues. I wonder whether explicitly casting action_queries to float32 could affect mixed-precision training or cause inconsistency with the official setup.

My current observations are:

Base VLM: Qwen3-VL-4B-Instruct
Dataset: LIBERO
Training config: official Quick Start configuration
Released checkpoint evaluation: works correctly
Self-trained checkpoint evaluation: success rate = 0
Training loss: decreases normally
Code modification:
    pred_actions = self.action_model.predict_action(action_queries.float())

Since the released checkpoint works correctly but my self-trained checkpoint completely fails, I suspect the issue may come from the training pipeline rather than the evaluation environment.

Could you please help clarify the following points?

Are the Quick Start training hyperparameters sufficient to reproduce a non-zero LIBERO success rate with Qwen3-VL-4B-Instruct?
Could the .float() cast on action_queries affect training under bf16/fp16 mixed precision?

Are there any hidden training details that are required but not mentioned in the Quick Start Guide?
Is there any required action normalization / unnormalization step during training or checkpoint evaluation?
Are there specific checkpoint-saving or checkpoint-loading rules for evaluating a self-trained model?
Should the training and evaluation configs exactly match in terms of action_mode, action_type, include_state, state_dim, and action_dim?
Could you provide the exact training and evaluation commands used to obtain the released LIBERO checkpoint?

The confusing part is that the loss decreases normally, but the learned policy achieves 0 success rate, while the released checkpoint works under the same evaluation setup.

**Comment by Cherishqx** [comment]
我也碰到了这个问题，请问是怎么解决的？

**Comment by BoZhiStudying233** [comment]
I encountered the same issue as well. Could you please let me know how you resolved it?

**Comment by yechunjin** [comment]
请问解决了吗？
